fontc/src/main.rs

```diff
@@ -52,34 +54,32 @@ impl Config {
     }
 }
 
-fn establish_build_dir() -> PathBuf {
-    let build_dir = Path::new("build");
-    if build_dir.exists() && !build_dir.is_dir() {
-        panic!("The name build is taken by something that isn't a directory");
+fn require_dir(dir: &Path) -> PathBuf {
```

Comment:
this could very reasonably return an error, I'm fine either way

Reply:
Hm. In re-reading I agree. Good catch.